### PR TITLE
Feature/49/add question detail page

### PR DESCRIPTION
**Pull request by syoung125**

### CODE REVIEW

**Comment by pickk-dev** [comment]
#### 📝Changed routes:


| [**/manage/question/\[id\]**](https://pickk-super-admin-9nbiknetc.vercel.app/manage/question/[id]) | [**/manage/question**](https://pickk-super-admin-9nbiknetc.vercel.app/manage/question) |
|:-:|:-:|
|<a href="https://pickk-super-admin-9nbiknetc.vercel.app/manage/question/[id]">Screenshot of /manage/question/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/L4EJiY2raowq5DmGhOAUnKMdVg4_.png">(view full size)</a> |<a href="https://pickk-super-admin-9nbiknetc.vercel.app/manage/question">Screenshot of /manage/question</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/Raeh_K8NMunwrnsmYRq6Bh1_JJM_.png">(view full size)</a> |







Commit c215513ba3c665fe4154b0d9f388c11c63fd4ca0 (https://pickk-super-admin-9nbiknetc.vercel.app).

File: src/components/question /detail/sections/item.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+export type ItemSectionProps = Pick<IItem, 'id' | 'name' | 'brand'>;
+
+function ItemSection({ id, name, brand }: ItemSectionProps) {
+  console.log(brand.nameKor);
+  return (
+    <DrawerSectionWrapper title="상품정보">
+      <BrandName>{brand.nameKor}</BrandName>
+      <ItemName>{name}</ItemName>
+      <ButtonWrapper>
+        <StyledA href={brand.siteUrl}>공홈링크</StyledA>
```
Comment (greatSumini) [inline src/components/question /detail/sections/item.tsx:18]:
```suggestion
        <StyledA href={purchaseUrl}>공홈링크</StyledA>
```

File: src/components/question /detail/sections/question.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  };
+
+  const handleOk = async () => {
+    setIsModalVisible(false);
+    await QuestionService.answerCreate(questionPk, { content: answer }).then(
+      () => {
+        message.info('답변이 등록되었습니다.');
+        setAnswer('');
+      }
+    );
```
Comment (greatSumini) [inline src/components/question /detail/sections/question.tsx:47]:
```suggestion
    await QuestionService.answerCreate(questionPk, { content: answer })
    message.info('답변이 등록되었습니다.')
    setAnswer('');
```

File: src/components/question /detail/index.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+function QuestionDetail({ id, question }: QuestionDetailProps) {
+  if (!question) return null;
+
+  return (
+    <Wrapper>
+      <ItemSection
+        id={question.item.id}
+        name={question.item.name}
+        brand={question.item.brand}
```
Comment (greatSumini) [inline src/components/question /detail/index.tsx:21]:
```suggestion
        {...question.item}
```

File: src/components/question /detail/index.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      />
+      <QuestionSection
+        title={question.title}
+        content={question.content}
+        createdAt={question.createdAt}
+        questionPk={id}
```
Comment (greatSumini) [inline src/components/question /detail/index.tsx:27]:
```suggestion
        {...question}
```

**Pull request merged**